scrapers/pantyjobs.yml

```
@@ -0,0 +1,25 @@
+name: pantyjobs
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - pantyjobs.com/scenes/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: '//div[@class=''col-md-12 aligncenter'']/h1'
```

you are using 2 `'` instead of `"`
the starting/ending  `'`  are not needed
```suggestion
        selector: //div[@class="col-md-12 aligncenter"]/h1
```